odoo/tests/common.py

```diff
@@ -1618,16 +1622,18 @@ def authenticate(self, user, password):
 
         return session
 
-    def browser_js(self, url_path, code, ready='', login=None, timeout=60, cookies=None, **kw):
+    def browser_js(self, url_path, code, ready='', login=None, timeout=60, cookies=None, watch=False, **kw):
```

Comment:
Having this as a paramater seems like a very bad idea, much too easy to leave enabled and start spawning un-managed Chrome instances on the runbot or wherever.

Reply:
what would you suggest? :)

Reply:
The only risk will be to start a useless chome process but in all case:
-> anyone may start a process manually
-> the docker will be killed at the end of the build, or timeout
-> the memory limit of the docker should preserv other running build 


**Using a kwarg** 
Pros:
- Easy to do inline when debugging a specific test
Cons
- May be forgotten and executed on runbot
- Warning may be missed if test is not executed

**Using a basic --dev flag (or any other command line flag)**
Pros:
- [x] Won't start on runbot 
Cons
- [ ] Will start for all tests

Other Solutions: 

Adding a linter may avoid missing it if the test is not executed, but if the test is executed, the linter won't do much: the build will fail anyway

Only allow watch based on other parameters (environment, positive test-tags, ...)

Having a way to watch a specific test using command line (maybe using test-tags). We already talked about that but this may be for another pr.

Reply:
> Cons
> * Will start for all tests

Except that when you activate the flag you most likely only run one or a few tests at a time with the `--test-tags`